spec/std/isa/ext/Zvqdotq.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Kallal Mukherjee.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../schemas/ext_schema.json
+
+$schema: "ext_schema.json#"
+kind: extension
+name: Zvqdotq
+type: unprivileged
+long_name: Vector quad widening 4D Dot Product 8-bit Integer
+company:
+  name: RISC-V International
+  url: https://riscv.org
+doc_license:
+  name: Creative Commons Attribution 4.0 International License
+  url: https://creativecommons.org/licenses/by/4.0/
+versions:
+  - version: "0.0.1"
+    state: unratified
+    ratification_date: null
+    contributors:
+      - name: Kenneth Dockser
+        email: kdockser@tenstorrent.com
+        company: Tenstorrent
+      - name: Kallal Mukherjee
+        email: ritamukherje62@gmail.com
+        company: Independent
```

Comment:
Unless you contributed meaningfully to the Zvqdotq extension, this should be removed.